spec/features/collection_objects_spec.rb

```diff
@@ -92,11 +92,10 @@
                 end
 
                 specify 'source' do
-                  s = FactoryBot.create(:valid_source, by: @user) 
-
+                  s = FactoryBot.create(:valid_source, by: @user)
                   fill_autocomplete('source_id_for_specimen_batchload', with: s.cached, select: s.id)
                   click_button('buf_preview')
-                  expect(page).to have_content("Using '#{s.cached}' as source.")
+                  expect(page).to have_content("Using '#{s.cached.gsub('<i>','').gsub('</i>', '')}' as source.")
```

Comment:
Why is gsub is necessary, did we get rid of `<i>`?

Reply:
not sure. This failed first time the changes were submitted, and I could not test feature-tests locally, had to guess what is is going on.

Reply:
can remove it and push again to see if it fails.

Reply:
New code introduced them in cached. On my computer I fixed this spec with `'#{s.cached.gsub(/<\/?i>/,'')}'` after lots of trials and errors because I wasn't noticing the actual problem. Then discarded my code and branched because the specs breakage was more extensive than this.

Reply:
Italics in cached are fine.  SInce this is feature spec this is OK, I see we are just matching the UI.